OrangeDesignSystem/Sources/OrangeDesignSystem/Modules/About/Internal/LitsItems/AboutListItems.swift

```diff
@@ -60,26 +60,22 @@ struct AboutListItems: View {
             Button {
                 action()
             } label: {
-                ODSListStandardItem(model: configuration.model)
+                ODSListItem(title: Text(configuration.title), leading: ODSListItem.Leading(icon: configuration.icon))
```

Comment:
Could be safer for the `ODSListItem` to expose _title_ parameter as a _String_ and not a _Text_.
If we need in the future to change the implementation of the `ODSListItem`or if we need to get rid of _Text_ we will have to break the API for this component. Maybe hiding _Text_ in the component _body_ implementation and expose _String_ will be lighter and more future-proof.

Reply:
why we should use an id of the text ?